unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatEntry.cs

```diff
@@ -222,8 +278,25 @@ public void SetFadeout(bool enabled)
     void Update()
     {
         Fade();
-
+        CheckHoverCoordinates();
         ProcessHoverPanelTimer();
+        ProcessHoverGotoPanelTimer();
+    }
+
+    private void CheckHoverCoordinates()
+    {
+        if (isOverCoordinates)
+            return;
+
+        int linkIndex = TMP_TextUtilities.FindIntersectingLink(body, Input.mousePosition, null);
+        if (linkIndex != -1)
```

Comment:
As before, you can early return here to avoid nesting.

Reply:
same as before

Reply:
In this case there's no code after this if.